packages/php-wasm/util/src/lib/php-vars.ts

```diff
@@ -2,11 +2,10 @@ const literal = Symbol('literal');
 
 export function phpVar(value: unknown): string {
 	if (typeof value === 'string') {
-		if (value.startsWith('$')) {
-			return value;
-		} else {
-			return JSON.stringify(value);
-		}
+		const escapedValue = JSON.stringify(value)
+			.replace(/\\/g, '\\\\')
+			.replace(/'/g, "\\'");
+		return `json_decode('${escapedValue}')`;
```

Comment:
I don't love this change, let's find a better way to turn a JavaScript string into a PHP string literal.

Reply:
I think that if we base64-encode it then we won't need to escape the JSON value.

```js
const escapedValue = btoa(JSON.stringify(value));
return `json_decode(base64_decode('${escapedValue}', true))`;
```

of course it also raises the question if we want to add `, /* associative */ true` to the PHP call to `json_decode()`. do we want an associate array or a `stdClass`?